Setup/Patch/Data/CreateCategoryAttribute.php

```diff
@@ -49,7 +49,8 @@ public function apply()
             'required' => false,//required for all customers
             'visible' => false,
             'system' => false,//to list the attribute in getCustomAttributes()
-            'backend' => ''
+            'backend' => '',
+            'global' => ScopedAttributeInterface::SCOPE_STORE,
```

Comment:
Don't we need here also `'default' => 0`?

Reply:
We will add it